test/cmdlineTests.sh

```diff
@@ -40,17 +41,13 @@ fi
 
 FULLARGS="--optimize --ignore-missing --combined-json abi,asm,ast,bin,bin-runtime,compact-format,devdoc,hashes,interface,metadata,opcodes,srcmap,srcmap-runtime,userdoc"
 
-## FUNCTIONS
-
-if [ "$CIRCLECI" ]
-then
-    function printTask() { echo "$(tput bold)$(tput setaf 2)$1$(tput setaf 7)"; }
-    function printError() { echo "$(tput setaf 1)$1$(tput setaf 7)"; }
-else
-    function printTask() { echo "$(tput bold)$(tput setaf 2)$1$(tput sgr0)"; }
-    function printError() { echo "$(tput setaf 1)$1$(tput sgr0)"; }
+# extend stack size in case we run via ASAN
```

Comment:
I think we should really only extend it when running ASAN. Otherwise we will run into stack problems in production that are not detected in the tests. But I'll still merge this for now to get it merged...